app/views/admin/shared/_tabnav_product.html.haml

```diff
@@ -15,7 +15,11 @@
   - if @product.is_a?(Instrument)
     = tab "Restrictions", edit_facility_price_group_product_path(current_facility, @product), (secondary_tab == "restrictions")
     = tab "Reservations", facility_instrument_schedule_path(current_facility, @product), (secondary_tab == "reservations")
-  = tab "Docs", [current_facility, @product, :file_uploads, file_type: "info"], (secondary_tab == "documentation")
+
+  = tab "Docs",
+    [current_facility, @product, :file_uploads, file_type: "info"],
```

Comment:
The "Docs" tab path needed to change from `/facilities/:id/:product_class/:id/files/upload?file_type=info` to `/facilities/:id/:product_class/:id/files?file_type=info`